doc/sphinx/Pacemaker_Explained/utilization.rst

```diff
@@ -98,167 +86,140 @@ tool can be used to set these attributes.
+really long time to solve".
 
-Clearly in a HA cluster, it's not acceptable to spend minutes, let alone hours
-or days, finding an optimal solution while services remain unavailable.
+In a high-availability cluster, it is unacceptable to spend minutes, let alone
+hours or days, finding an optimal solution while services are down.
 
-So instead of trying to solve the problem completely, Pacemaker uses a
-*best effort* algorithm for determining which node should host a particular
-service. This means it arrives at a solution much faster than traditional
-linear programming algorithms, but by doing so at the price of leaving some
-services stopped.
+Instead of trying to solve the problem completely, Pacemaker uses a "best
+effort" algorithm. This arrives at a quick solution, but at the cost of
+possibly leaving some resources stopped unnecessarily.
 
-In the contrived example at the start of this chapter:
+Using the example configuration at the start of this chapter:
 
 * ``rsc-small`` would be assigned to ``node1``
 
 * ``rsc-medium`` would be assigned to ``node2``
 
 * ``rsc-large`` would remain inactive
 
-Which is not ideal.
-
-There are various approaches to dealing with the limitations of
-pacemaker's placement strategy:
+That is not ideal. There are various approaches to dealing with the limitations
+of Pacemaker's placement strategy:
 
 * **Ensure you have sufficient physical capacity.**
 
-   It might sound obvious, but if the physical capacity of your nodes is (close to)
-   maxed out by the cluster under normal conditions, then failover isn't going to
-   go well. Even without the utilization feature, you'll start hitting timeouts and
-   getting secondary failures.
+   It might sound obvious, but if the physical capacity of your nodes is maxed
+   out even under normal conditions, failover isn't going to go well. Even
+   without the utilization feature, you'll start hitting timeouts and getting
+   secondary failures.
 
-* **Build some buffer into the capabilities advertised by the nodes.**
+* **Build some buffer into the capacities advertised by the nodes.**
 
-   Advertise slightly more resources than we physically have, on the (usually valid)
-   assumption that a resource will not use 100% of the configured amount of
-   CPU, memory and so forth *all* the time. This practice is sometimes called *overcommit*.
+   Advertise slightly more resources than we physically have, on the (usually
+   valid) assumption that resources will not always use 100% of their
+   configured utilization. This practice is sometimes called *overcommitting*.
 
 * **Specify resource priorities.**
 
-   If the cluster is going to sacrifice services, it should be the ones you care
-   about (comparatively) the least. Ensure that resource priorities are properly set
-   so that your most important resources are scheduled first. 
+   If the cluster is going to sacrifice services, it should be the ones you
+   care about the least.
```

Comment:
thinking out loud, constraints might sometimes be appropriate for helping deal with these limitations. In this simple case, you could solve the optimization problem yourself: if possible, `rsc-medium` should run on `node2`, and `rsc-small` and `rsc-large` should run on `node1`.

Not necessary to add anything further to this doc. You've done a lot in this PR and improved the docs quite a bit.

Reply:
Right, if someone is looking at utilization, they've scaled beyond simple constraints (the example is trivial for brevity)